### DISCUSSION TITLE
关于中译英 | English translation

### DISCUSSION THREAD

**Discussion by seguri** [doocs/advanced-java#66] [Ideas]
...or please make it easy to use with Google Translate:

![Screenshot_2019-05-14 Google Translate(1)](https://user-images.githubusercontent.com/2405157/57704863-4c3d6a80-7652-11e9-948c-fbcabc0eb458.png)

**Comment by yanglbme** [comment]
@seguri ...try this link https://doocs.github.io/advanced-java/#/docs/high-concurrency/mq-interview

**Comment by yanglbme** [comment]
@seguri You could copy and paste the link to your Google Chrome, when the page is fully loaded, click right and click “Translate to English” to translate this page. Hope the technical terms could make sense to you and feel free to raise up any questions.

**Comment by seguri** [comment]
> doocs.github.io/advanced-java/#/docs/high-concurrency/mq-interview

Doesn't work either.

> You could copy and paste the link to your Google Chrome, when the page is fully loaded, click right and click “Translate to English” to translate this page.

This works, thanks!

Do you have any plans to share for an English translation?

**Comment by yanglbme** [comment]
The translation work requires a lot of time and effort.

One's strength is limited, so I wonder if anyone is willing to work with me to complete the translation.

**Comment by codebuster007** [comment]
> The translation work requires a lot of time and effort.
> 
> One's strength is limited, so I wonder if anyone is willing to work with me to complete the translation.

Hi @yanglbme,
I'd like to work on this with you if it's not already in progress. I don't know Chinese but I intend to do translations with Google Translate 😅. If this is already in progress and need help, I'm willing to offer help.

cheers!

**Comment by ngarawakimani** [comment]
Someone please translate this repo

**Comment by codebuster007** [comment]
> 
> 
> > The translation work requires a lot of time and effort.
> > One's strength is limited, so I wonder if anyone is willing to work with me to complete the translation.
> 
> Hi @yanglbme,
> I'd like to work on this with you if it's not already in progress. I don't know Chinese but I intend to do translations with Google Translate 😅. If this is already in progress and need help, I'm willing to offer help.
> 
> cheers!


Hi @yanglbme , @seguri ,
I'm really interested in converting this project to English.
Please reach out if this is feasible.
Kind regards.
